plugins/CoreHome/vue/src/ReportingMenu/ReportingMenu.vue

```diff
@@ -283,10 +307,13 @@ export default defineComponent({
       NotificationsStore.remove(REPORTING_HELP_NOTIFICATION_ID);
 
       const isActive = ReportingMenuStoreInstance.toggleCategory(category);
-      if (isActive
-        && (category as SubcategoryContainer).subcategories
-        && (category as SubcategoryContainer).subcategories.length === 1
-      ) {
+
+      // one subcategory or a widget and some subcategories to allow to load the category
+      const { subcategories } = category as SubcategoryContainer;
+      const categoryCanLoad = (subcategories && subcategories.length === 1)
+        || (category.widget && subcategories && subcategories.length);
```

Comment:
I know I could extract the `subcategories` condition to the front though it seems better for legibility to keep them bunched like this.

Reply:
I'm wondering why in the second check ` || (category.widget && subcategories && subcategories.length);` 

that `subcategories && subcategories.length` is needed? Isn't there only being a widget enough?

Reply:
No, the widget itself doesn't do anything, it actually triggers the action of the first nested menu item. Nested menu items are subcategories, so we need at least one to be there (but can be multiple).

So the logic is:
- no widget override, exactly one subcategory -> we load the view for the first (only) item
- no widget override, more subcategories -> we open the nested menu and no change of view
- widget override, one or more subcategories -> we load the view for the first item

Reply:
Thanks for clarifying